### PR TITLE
Yuichiro PPP

### PR DESCRIPTION
**Pull request by yuifuku1118**

### CODE REVIEW

File: docs/team/yuifuku1118.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* **Documentation**:
+    * User Guide:
+        * Added documentation for the storage related features (i.e. `exit` and `clear`) [\#139]()
+  
+    * Developer Guide:
+        * Added implementation details of the `storage` component.[\#135]()
+        * Added the all the user stories   [\#35]()
+
+* **Community**:
+    * PRs reviewed (with non-trivial review comments): [\#119](), [\#120](), [\#51](), [\#50]()
```
Comment (ngchisern) [inline docs/team/yuifuku1118.md:32]: I believe you need to provide links to these pull requests as well so that they would work properly.

**Review by ngchisern — CHANGES_REQUESTED** [review]
Provided a suggestion below.

File: docs/team/yuifuku1118.md (hunk 1 above)
Comment (yuifuku1118) [reply docs/team/yuifuku1118.md:32]: Oh yes, thanks for commenting! Will fix that right away.

**Review by ngchisern — APPROVED** [review]
LGTM

**Pull request merged**